charts/dapr/charts/dapr_placement/templates/dapr_placement_service.yaml

```diff
@@ -2,10 +2,17 @@ kind: Service
 apiVersion: v1
 metadata:
   name: dapr-placement
+  labels:
+    app: dapr-placement
 spec:
   selector:
     app: dapr-placement
+  # placement must be able to resolve pod address to join initial cluster peers
+  # before POD is ready
+  publishNotReadyAddresses: true
   ports:
-  - protocol: TCP
-    port: {{ .Values.ports.port }} 
```

Comment:
headless service doesn't support `targetPort` so we need to change the placement port from 80 to 50005